### ISSUE TITLE
Dataform encountered an error: Unexpected token (2:15)

### ISSUE THREAD

**Issue by ldolberg** [dataform-co/dataform#1305]
Hey @BenBirt @josiehall @lewish @canberkkoparal 

We are having an issue running Dataform using version 1.20 and 1.21. We were able to reproduce the issue from `ubuntu:latest` and `python:3.8-slim` as well as in github actions.

Could you please advise? This issue is impacting us pretty heavily since we are unable to run our data transformations.

The full stack trace is 
```
Dataform encountered an error: Unexpected token (2:15)
SyntaxError: Unexpected token (2:15)
    at Parser.pp$4.raise (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:3460:15)
    at Parser.pp$9.unexpected (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:756:10)
    at Parser.pp$9.semicolon (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:733:68)
    at Parser.pp$8.parseExpressionStatement (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:1216:10)
    at Parser.pp$8.parseStatement (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:949:26)
    at Parser.pp$8.parseBlock (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:1232:23)
    at Parser.pp$8.parseStatement (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:914:38)
    at Parser.pp$8.parseTopLevel (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:813:23)
    at Parser.parse (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:586:17)
    at Function.parse (/usr/local/nvm/versions/node/v17.3.0/lib/node_modules/@dataform/cli/node_modules/acorn/dist/acorn.js:636:37)
```

Please find attached the Dockerfile we use to reproduce the issue.
```Docker
FROM ubuntu:latest
ENV NODE_VERSION=17.3.0 \
    NVM_DIR=/usr/local/nvm

ENV PATH="$POETRY_HOME/bin:$VENV_PATH/bin:$NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH"

RUN apt update && apt install curl -y
SHELL ["/bin/bash", "-o", "pipefail", "-c"]

RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash

# install node and npm
RUN source "$NVM_DIR/nvm.sh" \
    && nvm install "$NODE_VERSION" \
    && nvm alias default "$NODE_VERSION" \
    && nvm use default

RUN npm i -g @dataform/cli@1.20.0

RUN mkdir dataform_test
WORKDIR dataform_test
RUN dataform init bigquery --default-database test
RUN dataform compile
```

**Comment by GJMcGowan** [comment]
Thank you for raising this issue. I have confirmed it locally and raised with the engineering team. 

We are looking into it and will get back to you with an update tomorrow.

**Comment by GJMcGowan** [comment]
This issue has been resolved in version`1.21.1`, please upgrade.

Resolved here: https://github.com/dataform-co/dataform/pull/1306

Thank you for raising this!
